src/ajax.js

```diff
@@ -33,6 +33,12 @@ export function ajaxBuilder(timeout = 3000, {request, done} = {}) {
         }
       };
 
+      if (typeof callbacks.timeout !== 'function') {
+        callbacks.timeout = function (e) {
+          utils.logError('xhr timeout after ', x.timeout, 'ms');
```

Comment:
side effect here is the logError isn't logged if you define a custom timeout function.

Reply:
@mkendall07 That is true. What are you suggesting? Do some sort of extension of the previous function ? like

```x.timeout = function (e) {
    callbacks.timeout(e);     
    utils.logError('xhr timeout after ', x.timeout, 'ms');
}```

What if the dev would want to "just" extend the logError to display real timeout time (not the x.timeout settings) ? Than he would always have two messages in console.

Reply:
@mkendall07 Hello, can I please get any kind of instruction? :) Thank you

Reply:
this was just an observation, not sure you need to do anything about it. Can you please update the usage with a JSdoc notation?